docs/UserGuide.md

```diff
@@ -135,7 +135,7 @@ Format: `edit INDEX [n/NAME] [e/Email] [s/STUDENT_ID] [N/NUSNet_ID] [g/GITHUB_ID
     specifying any tags after it.
 
 Examples:
-* `edit 1 p/91234567 e/e0123456@u.nus.edu` Edits the phone number and email address of the 1st person to be `91234567` and `e0123456@u.nus.edu` respectively.
+* `edit 1 p/91234567` Edits the phone number of the 1st person to be `91234567`.
 * `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
 * `edit 1 n/Siddharth Srivastava t/IncompleteIP` Edits the name of the 1st person to be `Siddharth Srivastava` and replaces all existing tags with the tag `IncompleteIP`.
```

Comment:
Can consider adding an example showing that changing the NUSnet ID will also change the email

Reply:
Requisite changes made